src/Tasks/dependency-check-build-task/package.json

```diff
@@ -11,6 +11,8 @@
   "dependencies": {
     "azure-pipelines-task-lib": "^3.0.6-preview.1",
     "decompress-zip": "^0.3.3",
+    "is-core-module": "^2.4.0",
+    "resolve": "^1.20.0",
```

Comment:
Minimum version 1.20.0 is needed for is-core-module because at least one previous version would break the extension (file type in extension manifest would have unallowed character).